lib/Atat/Objects.pm

```diff
@@ -0,0 +1,96 @@
+# Domain model.
+#
+# This file describes our domain using an object model.
+#
+# The following terminology may be helpful:
+#     * 'package' defines a class.
+#     * A class 'has' an attribute, which may be another object (delegation).
+#     * Plural attributes imply a 1-many (or many-many) relationship.
+#     * A class 'is' another class (vertical composition).
+#     * A class 'does' a role (horizontal composition).
+
+# User: An ATAT user.
```

Comment:
Definitions:   
- A user is uniquely identified by their email, and authenticated by their CAC 
- A user can be any DoD employee or contractor

- cards

 user can be any

Reply:
They will _not_ be uniquely identified by their email (although that should be a unique field). We will need a permanent identifier outside of email address as that can change.